### ISSUE TITLE
Not working with a brand new device

### ISSUE THREAD

**Issue by poussun-ops** [benisai/Firewalla-USB-WiFi-MAC-Updater#1]
I tried the script with some new 8811CU devices, and, even if the mac adresses are modified and the devices recognized by the app, scanning the network fails... Just Fyi, as you asked for :)

**Comment by davelee** [comment]
+1

**Comment by benisai** [comment]
Yeah I have a feeling Firewalla added additional checks I suspect. So much for being an open source company. 

I haven't used Wi-Fi for over a year so I haven't actually checked the script.
